app/controllers/user.js

```diff
@@ -0,0 +1,174 @@
+const jwt = require('jsonwebtoken');
+const model = require('../models');
+
+const secret = process.env.SECRET_TOKEN || 'secret';
+
+// To create text-friendly user details
+const formattedUser = (user) => {
+  const userDetails = {
```

Comment:
You can use destructuring to make this cooler. Read it up [here](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)